From c4f358e51b935354b97dbec887d5fc2a2b39eab2 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 6 Sep 2022 07:43:11 +0800
Subject: [PATCH 05/60] Revert "drm/bridge: tc358767: Fix (e)DP bridge endpoint
 parsing in dedicated function"

This reverts commit 3422e24af9ba79eb4ba70646dfd3c1621ea92558.
---
 drivers/gpu/drm/bridge/tc358767.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/tc358767.c b/drivers/gpu/drm/bridge/tc358767.c
index b4f7e7a7f7c51..3e365cd7d0a34 100644
--- a/drivers/gpu/drm/bridge/tc358767.c
+++ b/drivers/gpu/drm/bridge/tc358767.c
@@ -1559,7 +1559,7 @@ static int tc_probe_edp_bridge_endpoint(struct tc_data *tc)
 		tc->bridge.type = DRM_MODE_CONNECTOR_DisplayPort;
 	}
 
-	return 0;
+	return ret;
 }
 
 static void tc_clk_disable(void *data)
-- 
2.25.1

